### PR TITLE
Modified StandWithUkraine PR

### PR DESCRIPTION
**Pull request by Trott**
This takes https://github.com/nodejs/nodejs.org/pull/4449 and simplifies it so that we can land it immediately. We have the option to adjust it to have more content later, if desired.

### CODE REVIEW

**Comment by nschonni** [comment]
Wouldn't the easiest way just to use our existing banner infra instead of creating a new set?

**Comment by Trott** [comment]
> Wouldn't the easiest way just to use our existing banner infra instead of creating a new set?

Sure, we can do that.

**Comment by Trott** [comment]
> > Wouldn't the easiest way just to use our existing banner infra instead of creating a new set?
> 
> Sure, we can do that.

OK, done.

**Review by alexandrtovmach — APPROVED** [review]

**Comment by alexandrtovmach** [comment]
I've decided to add another page to provide more details about situation, but agree, that better to publish at least something immediately, and then we can move forward with more details.

**Comment by alexandrtovmach** [comment]
@Trott should we wait more for 24hours? or we can publish it? I don't see any objections by anyone, just some improvements and suggestions, so maybe it makes sense to release and to have something that help us now?

**Review by aduh95 — APPROVED** [review]

**Review by RaisinTen — APPROVED** [review]

**Review by mpelekh — APPROVED** [review]

**Review by BridgeAR — APPROVED** [review]

**Review by fhinkel — CHANGES_REQUESTED** [review]
I'm deeply upset about the tragic, unlawful and unjustified invasion of Ukraine. As an organization though, we can not and should not solicit donations. Thanks for understanding.

**Comment by alexandrtovmach** [comment]
@fhinkel could you reference to this rule somewhere in code of conduct or some policy?

**Comment by alexandrtovmach** [comment]
@fhinkel please?

**Review by jasnell — APPROVED** [review]

**Comment by alexandrtovmach** [comment]
@fhinkel your position sounds opinionated without proofs from code of conduct or other community policy. Maybe someone from @nodejs/tsc group can say something?

Google, Facebook, Apple – already show their support, so why Node.js can't? All world showing support to Ukraine, and I think Node.js community can show their support too.
https://www.youtube.com/watch?v=r0FuHDeIZZM&ab_channel=NBCNews

Adding a link to Red Cross (non-profit volunteering org) donations, it's a small and easy step that we can do here and now. Also, @benjamingr suggested [other steps](https://github.com/nodejs/nodejs.org/pull/4446#issuecomment-1054596689) that can be done later

**Comment by fhinkel** [comment]
Sorry for not getting back to you immediately. We're not allowed to raise money on behalf of the OpenJS Foundation, see https://openjsf.org/wp-content/uploads/sites/84/2021/01/OpenJS-Foundation-Trademark-Policy-2021-01-12.docx.pdf

> Maybe someone from https://github.com/orgs/nodejs/teams/tsc group can say something?

I'm a TSC member.

**Comment by Trott** [comment]
> Sorry for not getting back to you immediately. We're not allowed to raise money on behalf of the OpenJS Foundation, see https://openjsf.org/wp-content/uploads/sites/84/2021/01/OpenJS-Foundation-Trademark-Policy-2021-01-12.docx.pdf

That refers to logo use for crowdfunding. I don't think it applies in this case but it's entirely possible I'm wrong about that. Let's ping @brianwarner from the Foundation so he can weigh in.

**Comment by Trott** [comment]
The OpenJS Foundation folks have provided wording, which I will put into our banner for this PR.

Here's the wording:

> We stand with the people of Ukraine. We encourage compassion, and hope for peace. Please support humanitarian efforts for the Ukraine crisis through the [International Committee of the Red Cross](https://www.icrc.org/en/donate/ukraine). #StandWithUkraine

That should be on the OpenJS Foundation web page soon.

Looping in @rginn in case I've misrepresented anything.

**Comment by rginn** [comment]
LGTM <3 

Thanks Rich @Trott.

**Review by fhinkel — APPROVED** [review]

**Review by danielleadams — APPROVED** [review]

**Comment by alexandrtovmach** [comment]
Thanks everyone for seeking consensus

**Comment by alexandrtovmach** [comment]
@Trott would you change this message in banner? Because if I'm going to create it from my side, another PR merge process will take another amount of time

**Comment by Trott** [comment]
> @Trott would you change this message in banner? Because if I'm going to create it from my side, another PR merge process will take another amount of time

Yes, working on it right now.

File: locale/en/site.json [hunk 1]
```diff
@@ -163,15 +163,10 @@
   },
   "banners": {
     "index": {
-      "startDate": "2022-01-04T16:00:00.000Z",
```
Comment (nschonni) [inline locale/en/site.json:166]: @Trott as you're rebasing for the merge issue, I think you only need to change these 4 lines and the rest can be reverted

**Comment by Trott** [comment]
My implementation could no doubt use some refining--I have a `style` attribute, for example--but I think this is good enough for an initial launch and people can improve the CSS (or not) afterwards.

image

**Comment by alexandrtovmach** [comment]
LGTM, thanks

File: locale/en/site.json [hunk 2]
```diff
@@ -163,15 +163,11 @@
...
-      "text": "New security releases now available for Node.js 12, 14, 16, and 17 release lines",
-      "link": "https://nodejs.org/en/blog/vulnerability/jan-2022-security-releases/"
-    },
-    "blacklivesmatter": {
-      "visible": false,
-      "text": "#BlackLivesMatter",
-      "link": "black-lives-matter/"
+      "startDate": "2022-02-28T16:00:00.000Z",
+      "endDate": "2022-03-21T16:00:00.000Z",
+      "rawHTML": "<p class=\"home-version home-version-banner\" style=\"font-size: large; text-align: left; line-height: 1.25; \">We stand with the people of Ukraine. We encourage compassion, and hope for peace. Please support humanitarian efforts for the Ukraine crisis through the <a href=\"https://www.icrc.org/en/donate/ukraine\">International Committee of the Red Cross</a>. #StandWithUkraine</p>",
```
Comment (tniessen) [inline locale/en/site.json]: My understanding of the style guide, not sure if preferred by native speakers:

```suggestion
      "rawHTML": "<p class=\"home-version home-version-banner\" style=\"font-size: large; text-align: left; line-height: 1.25; \">We stand with the people of Ukraine. We encourage compassion and hope for peace. Please support humanitarian efforts for the Ukraine crisis through the <a href=\"https://www.icrc.org/en/donate/ukraine\">International Committee of the Red Cross</a>. #StandWithUkraine</p>",
```

**Review by tniessen — APPROVED** [review]

File: locale/en/site.json (hunk 2 above)
Comment (Trott) [reply locale/en/site.json]: I would leave out the comma too. However, I'm inclined to use the OpenJS text verbatim. Unless we see it on the OpenJS website without the comma, I'm inclined to keep it.

Comment (tniessen) [reply locale/en/site.json]: That makes a lot of sense! :)

Comment (rginn) [reply locale/en/site.json]: I'm totally fine removing the comma and will remove it from our text. We rarely have consensus on commas at OpenJS. :)

Comment (Trott) [reply locale/en/site.json]: Tired: Debating semicolons, tabs, spaces...

Wired: Commas! Let's debate commas!

**Comment by Trott** [comment]
The banner on openjsf.org is live.

image

So I'm going to land this now.

**Comment by wangyiliangnet** [comment]
Why dont you stand with Yemen, Syrian or Iraq when their people attacked, bombed. 

NODE is owned by the OpenJS Foundation, not for your own politics tricks under rediculous two sets of rules. 

How dare you guys represent the whole foundation to publish such a  politic statement. SHAMELESS!

image
image

**Comment by indexzero** [comment]
Open Source is made by people.

By and large, many of those people have broad choice in what contributions they get to spend their time on.

I believe (but cannot say definitively in any official capacity) that had a PR to this repository been opened when those atrocities were at their peak which made a similar statement standing with those actively endangered & oppressed that PR would have been swiftly merged and released.

**Comment by xiaochenggit** [comment]
fuck

**Comment by xiaochenggit** [comment]
говорить как чахотка

**Comment by wangyiliangnet** [comment]
> 

So can you explain why we did't see any statement to support people in Yemen, Syrian or Iraq? Are't they "actively endangered & oppressed " by the western invaders?

**Pull request merged**